test/working.test.js

```diff
@@ -1,27 +1,92 @@
-const test = require('ava');
-const { ServiceBroker } = require('moleculer');
-const fetch = require('node-fetch');
-const HealthMiddleware = require('..');
+const test = require("ava");
```

Comment:
No reformat in this pull request.
Keep focused on the feature added.
By the way, we prefer `'` over `"` for strings.